### PR TITLE
Drop support for Ruby 2.0

### PR DESCRIPTION
**Pull request by joneslee85**
Sadly, it has reached EOL

### CODE REVIEW

**Comment by codyolsen** [comment]
@joneslee85 It's been almost 5 months since a version bump and there are quite a few features that were added since then. How would you feel if we did a version bump and THEN dropping Ruby 2.0 as the very next commit? That way we still give legacy users the new features added since the last version.

**Comment by joneslee85** [comment]
@codyolsen that's a good idea, I leave it to @jeremytregunna :)

**Comment by jeremytregunna** [comment]
I'm all for this. I'm in meetings all day, but as soon as I have 30 minutes, I'll get it done today.

**Comment by joneslee85** [comment]
@jeremytregunna do you want me to take care of this on your behalf?

File: README.md [hunk 1]
```diff
@@ -20,7 +20,10 @@ Seriously, [check it out](http://www.trello.com/).
 Full Disclosure: This library is mostly complete, if you do find anything missing or not functioning as you expect it
 to, please [let us know](https://trello.com/card/spot-a-bug-report-it/4f092b2ee23cb6fe6d1aaabd/17).
 
-Supports Ruby 2.0 or newer. Version 1.3.0 is the last version that supports Ruby that is older than 1.9.3. 
+Supports Ruby 2.1.0 or newer.
+
+Version 1.3.0 is the last version that supports Ruby that is older than 1.9.3. 
+Version 1.4.1 is the last version that supports Ruby that is newer than 1.9.3 and older than 2.0.
```
Comment (jeroenj) [inline README.md]: Fyi: 1.4.2 has already been released so this notice should be a different version.

It's also stating incorrectly that v1.4 is the last version to support a Ruby older than 2.0. It's actually the last version that does support ruby 2.0.

I'd change it to something like this:

* Use version 1.3.0 or earlier for ruby 1.9.3 support.
* Use version 1.4.x or earlier for ruby 2.0.0 support.

**Comment by jeremytregunna** [comment]
@joneslee85 Yes please

**Pull request merged**